cryptography/src/commonMain/kotlin/com/wire/kalium/cryptography/MLSClient.kt

```diff
@@ -8,6 +8,16 @@ typealias MLSKeyPackage = ByteArray
 
 interface MLSClient {
 
+    /**
+     * Delete local MLS DB and files
+     *
+     * @return true when delete is successful and false otherwise
+     */
+    fun clearLocalFiles(): Boolean
+
+
+    fun close()
```

Comment:
What is close meant to do?

Can we document this?

Reply:
As I understand from the MLS docs, It's used to release any locks the C code have on the MLS resources, aka the DB.

Even though close is called, there are still some files that are not cleared, mainly the `db-wal` and `db-shm` I will ask  Mathieu about this issue.

But the logout is working correctly, but can't confirm if MLS DB state is valid after the login (login after log out)

Reply:
I guess I mean a bit different.

Why would I call close and not deleteFiles for example?

Reply:
Good question, I guess it's only needed when deleting the files, soft logout will keep the files -> the locks can stay

Reply:
I will change it and make close part of `deleteLocalFiles` function (I will do the same for `ProteusClient`)